scripts/verify-contracts.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+import ape
+import click
+import structlog
+from eth_utils import to_checksum_address
+import beamer.deploy.artifacts
+import beamer.deploy.config
+from beamer.typing import ChainId
+
+
+log = structlog.get_logger(__name__)
+
+
+def _get_contract_info(artifacts_dir: Path) -> dict[ChainId, dict[str, dict[str, str]]]:
+    contract_info: dict[ChainId, dict[str, dict[str, str]]] = {}
+    for file_path in artifacts_dir.glob("*.deployment.json"):
+        deployment = beamer.deploy.artifacts.Deployment.from_file(file_path)
+        assert deployment.chain is not None
```

Comment:
This is probably going to fail for `base.deployment.json`.

Reply:
yes fixed it now